.NET/Microsoft.Recognizers.Definitions.Common/French/DateTimeDefinitions.cs

```diff
@@ -720,11 +721,20 @@ public static class DateTimeDefinitions
       public const string NightRegex = @"\b(minuit|nuit)\b";
       public static readonly Dictionary<string, int> WrittenDecades = new Dictionary<string, int>
         {
-            { @"", 0 }
+            { @"cent", 0 },
```

Comment:
Why is this falling in the decades config?

Reply:
In other language like English, the hundreds is also in the WrittenDecades.

Reply:
The "cent" entry is unnecessary, I will remove it.

Reply:
> In other language like English, the hundreds is also in the WrittenDecades.

The "cent" entry is unnecessary, I will remove it.